study-builder/studies/rgp/i18n/en.conf

```diff
@@ -116,127 +121,88 @@
   condition?""",
   "website_prompt_optional": "(optional)",
   "website_placeholder": "If yes, provide the link here",
```

Comment:
Missed this in the previous version: there should be a period after "provide the link here"